pkg/noderesources/resource_allocation.go

```diff
@@ -73,18 +73,18 @@ func (r *resourceAllocationScorer) score(
 	}
 	if klog.V(10).Enabled() {
 		if len(pod.Spec.Volumes) >= 0 && utilfeature.DefaultFeatureGate.Enabled(features.BalanceAttachedNodeVolumes) && nodeInfo.TransientInfo != nil {
-			klog.Infof(
-				"%v -> %v: %v, map of allocatable resources %v, map of requested resources %v , allocatable volumes %d, requested volumes %d, score %d",
-				pod.Name, node.Name, r.Name,
-				allocatable, requested, nodeInfo.TransientInfo.TransNodeInfo.AllocatableVolumesCount,
-				nodeInfo.TransientInfo.TransNodeInfo.RequestedVolumes,
-				score,
+			klog.InfoS(
+				"score resources and volumes",
```

Comment:
I have no idea about what this log's message should be. Can you give me some suggestions? @Huang-Wei